ssclj/jar/pom.xml

```diff
@@ -163,6 +163,11 @@
       <artifactId>expectations</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>honeysql</groupId>
+      <artifactId>honeysql</artifactId>
+      <version>0.5.2</version>
```

Comment:
Please remove the explicit version from here.  I've added it to the main `pom.xml` file and pushed this into master.  Are `korma` and `honeysql` complementary or would it make sense to phase out `korma` use?